### ISSUE TITLE
Hippocampal sclerosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7771]
**Preferred term label:**
Hippocampal sclerosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Hippocampal sclerosis (HS) is a neuropathologic finding characterized by neuronal loss and gliosis in the CA-1 and subiculum of the hippocampus. PMID: 12117357

Hippocampal sclerosis is now recognized as one of the
main causes of focal epilepsy and is present in approximately
10% of adults with new-onset focal epilepsy.3 
PMID: 26060898
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
updating definition
